docs/developer-guide/developer.md

```
@@ -40,6 +40,29 @@ the `types.go` has been created according to sig-apimachinery docs, the develope
 CRD along with the deep-copy methods and actual yaml files which get created in `_output/crd`
 folder and are copied over to `helm/ovn-kubernetes/crds` to then be used when creating a KIND cluster.
 
+## Generating and Updating Mocks
+
+Mocks for unit tests are auto-generated using [mockery](https://github.com/vektra/mockery)
+and configured via `go-controller/.mockery.yaml`. To regenerate all mocks:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)